server/execution/api/patient.py

```diff
@@ -51,8 +51,11 @@ def get_all_patient():
     """ Returns all patients stored in the scenario. """
     try:
         execution, _ = util.get_execution_and_player()
+        patient_ids = list(execution.scenario.patients.keys())
         return {
-            "patient_ids": list(execution.scenario.patients.keys())
+            "patient_ids": patient_ids,
+            "patient_names": [execution.scenario.patients[patient_id].name for
+                              patient_id in patient_ids]
```

Comment:
I'd suggest something like 
```python
        return {
            "patient_id": patient.id,
            "patient_name": patient.name
        } for patient in execution.scenario.patients.values() ]
```  
I do think this is cleaner in code and also semantically, but not sure if this would be worth the trouble as you'd have to change the class in the app..

Reply:
Your approach is cleaner, but it introduces incompatibility with older versions and requires additional server modifications, including updates to tests and other